### DISCUSSION TITLE
Is it possible to get custom maps on the web version?

### DISCUSSION THREAD

**Discussion by YourAverageNZPlayer** [nzp-team/nzportable#901] [General]
I want to try a custom map with my boy, but he only can use the web version and I don't know if it is possible to get them on the web version.

**Comment by FrostyXPP** [comment]
I have been wondering that too

**Comment by qloakongithub** [comment]
Yes. Take the pk3 file (if it's zip rename it to pk3) and drag it into the website.

**Reply by spectrum1-wave** [reply]
Instructions unclear. I ended up with a plunger stuck in my toilet.

**Comment by spectrum1-wave** [comment]
[Screen recording 2025-03-24 8.40.48 PM.webm](https://github.com/user-attachments/assets/46e910c9-6503-4e83-83ba-117d75fab815)

Sorry if its a little bit chopped. My work computer simply is not the best but hope it helps

**Reply by spectrum1-wave** [reply]
If you get lucky enough you might find it in coop users' maps
